.github/workflows/docker-build.yml

```diff
@@ -96,6 +96,7 @@ jobs:
           labels: ${{ steps.meta-ort-minimal.outputs.labels }}
           target: minimal
           cache-from: type=registry,ref=${{ env.REGISTRY }}/${{ github.repository_owner }}/ort:cache
+          cache-to: type=registry,ref=${{ env.REGISTRY }}/${{ github.repository_owner }}/ort:cache,mode=max
```

Comment:
That was left out on purpose, all layers from the ort-minimal image are contained in the full ort image built before, so using the cache of the full image here in cache-from makes the build really fast. Writing the cache would overwrite the cache image with just the layers from the minimal cache. I thought I had documented that, but I can't find it anywhere, so maybe add an explanatory comment instead.

Reply:
I've added a comment instead.

Reply:
Ah, right. Thanks, fixed.